android/build-commissioner-libs.sh

```diff
@@ -27,7 +27,13 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-readonly CUR_DIR="$(dirname "$(realpath -s "$0")")"
+if [ "$(uname)" = "Linux" ]; then
+    echo "OS is Linux"
+    readonly CUR_DIR="$(dirname "$(realpath -s "$0")")"
```

Comment:
Looking at the "-s" option of the realpath command, I think it's okay to remove "-s" on both Linux and Macos.

```
➜  ~ realpath --help
Usage: realpath [OPTION]... FILE...
Print the resolved absolute file name;
all but the last component must exist

  -e, --canonicalize-existing  all components of the path must exist
  -m, --canonicalize-missing   no path components need exist or be a directory
  -L, --logical                resolve '..' components before symlinks
  -P, --physical               resolve symlinks as encountered (default)
  -q, --quiet                  suppress most error messages
      --relative-to=DIR        print the resolved path relative to DIR
      --relative-base=DIR      print absolute paths unless paths below DIR
  -s, --strip, --no-symlinks   don't expand symlinks
  -z, --zero                   end each output line with NUL, not newline
      --help        display this help and exit
      --version     output version information and exit
```

Reply:
done